scripts/enhance-requires

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+version_deps () {
+	while read dep
+	do
+		if [[ "${dep}" =~ ^[^\(]*\(\) ]] && rpm -q --whatprovides "${dep}" &> /dev/null
+		then
+			printf "($dep with %s)\n" "$(rpm -q --whatprovides "${dep}" --qf '%{NAME} >= %{VERSION}')"
+		else
+			printf "%s\n" "${dep}"
+		fi
+	done
+}
+
+/usr/lib/rpm/find-requires | version_deps
```

Comment:
Shouldn't this be %{_rpmconfigdir}/find-requires, i.e. without a hardcoded path?

Without `set -o pipefail`, this will ignore the return value from `find-requires`, which seems iffy.

Reply:
> Shouldn't this be %{_rpmconfigdir}/find-requires, i.e. without a hardcoded path?

Is the content of that macro available to generator scripts?

Reply:
The find-requires script, for example, does the same thing.   It calls "/usr/lib/rpm/rpmdeps".